infra/scripts/local-ci.sh

```diff
@@ -0,0 +1,198 @@
+  GetGovcParamsFromEnv
+elif [[ -z $secretsfile ]] ; then
+  secretsfile=${SECRETS_FILE}
+  GetParamsFromSecrets
+else
+  GetParamsFromSecrets
+fi
+
+if [[ -z "${TARGET_VCH}" ]] && [[ -z $target_vch ]] && [[ -z $secrets_target_vch ]]; then 
+  usage
+elif [[ -z $target_vch ]] ; then
+  target_vch=${TARGET_VCH}
+fi
+
+if [[ -z "${DOCKER_TEST}" ]] && [[ -z $docker_test ]] && [[ -z $secrets_docker_test ]] ; then
+  usage
+elif [[ ! -z "${DOCKER_TEST}" ]] ; then
+  docker_test=${DOCKER_TEST}
+elif [[ -z $docker_test ]] ; then
+  docker_test=$secrets_docker_test
+fi
+
+if [[ -z "${GITHUB_API_KEY}" ]] && [[ -z $github_api_key ]] && [[ -z $secrets_api_key ]] ; then
+  usage
+elif [[ ! -z "${GITHUB_API_KEY}" ]] ; then
+  github_api_key=${GITHUB_API_KEY}
+elif [[ -z $github_api_key ]] ; then
+  github_api_key=$secrets_api_key
+fi
+
+if [[ -z "${TEST_URL}" ]] && [[ -z $test_url ]] && [[ -z $secrets_url ]] ; then
+  usage
+elif [[ ! -z "${TEST_URL}" ]] ; then
+  test_url=${TEST_URL}
+elif [[ -z $test_url ]] ; then
+  test_url=$secrets_url
+fi
+
+if [[ -z "${TEST_DATASTORE}" ]] && [[ -z $test_datastore ]] && [[ -z $secrets_datastore ]] ; then
+  usage
+elif [[ ! -z "${TEST_DATASTORE}" ]] ; then
+  test_datastore=${TEST_DATASTORE}
+elif [[ -z $test_datastore ]] ; then
+  test_datastore=$secrets_datastore
+fi
+
+if [[ -z "${TEST_USERNAME}" ]] && [[ -z $test_username ]] && [[ -z $secrets_username ]] ; then
+  usage
+elif [[ ! -z "${TEST_USERNAME}" ]] ; then
+  test_username=${TEST_USERNAME}
+elif [[ -z $test_username ]] ; then
+  test_username=$secrets_username
+fi
+
+if [[ -z "${TEST_PASSWORD}" ]] && [[ -z $test_password ]] && [[ -z $secrets_password ]] ; then
+  usage
+elif [[ ! -z "${TEST_PASSWORD}" ]] ; then
+  test_password=${TEST_PASSWORD}
+elif [[ -z $test_password ]] ; then
+  test_password=$secrets_password
+fi
+
+##### Do the work
+mkdir -p $odir
+
+testsContainer=$(docker create -it \
+                        -w /vic \
+                        -v "$vic_dir:/vic" \
+                        -v "${odir}:/tmp" \
+                        -e GOVC_URL="$ip" \
+                        -e GOVC_INSECURE=1 \
+                        -e GITHUB_AUTOMATION_API_KEY=${github_api_key}\
+                        -e TEST_URL_ARRAY=${test_url}\
+                        -e TEST_DATASTORE=${test_datastore}\
+                        -e TEST_USERNAME=${test_username}\
+                        -e TEST_PASSWORD=${test_password}\
+                        -e TARGET_VCH=${target_vch}\
+                        -e DEBUG_VCH=1\
+                        ${ci_container}\
+                        bash -c "pybot -d /vic/${odir} /vic/tests/test-cases/"$docker_test";\
+                 mv *-container-logs.zip /tmp/ 2>/dev/null; \
+                 mv VCH-*-vmware.log /tmp/ 2>/dev/null; \
+                 mv vic-machine.log /tmp/ 2>/dev/null; \
+                 mv index.html* /tmp/ 2>/dev/null; \
+                 mv VCH-* /tmp/ 2>/dev/null")
```

Comment:
surprised to see these here - suggests to me that either I don't understand what the `pybot -d` option does, or it's not taken proper effect.